### PR TITLE
InfromationBoxComponent_add

### LINKED ISSUE #17: [Component] Create a reusable InformationBox Component            (Hero Section 4/n)
### Description

Create a reusable component of InformationBox

This component will accept following arguments/props
- Title
- Icon/Image
- InformationText

The component might be structured like this:
```
import React from 'react'

const InformationBox = ({title, imagePath, description}) => {
  return (
    <div>
      
    </div>
  )
}

export default InformationBox
```

- You can store image in `/src/components/assets` folder and then we can pass the path of this image to the `imagePath`

Relevant ScreenShot/Images

image

(Image referred above is at the center circle)

### References/Useful links:

- [https://www.freecodecamp.org/news/how-to-build-reusable-react-components/]()

### PR DESCRIPTION
**Pull request by vrunda2005**
# Welcome to the PClub WOC 4.0 Project! 👋🎉

Thanks for taking the time to fill out this Pull Request Template.
---

## Pull Request Title
[Component] Create a reusable InformationBox Component (Hero Section 4/n) #17 

## Related Issue
Solves/Resolves/Fixes #17.

## Description
-Added a reusable component called InformationBox.
-Styled the component to display a title, an icon, and a description in a clean and simple layout.
-This component can be used multiple times wherever needed.

## Checklist
Please ensure the following before submitting the PR:
- [yes ] I have read the `CODE_OF_CONDUCT.md` file and followed its guidelines.
- [yes ] I have followed the design specified in the Figma file (if applicable).

### CODE REVIEW

File: src/components/InformationBoxComponent/InformationBoxComponent.jsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      <div style={styles.iconContainer}>
+        <img src={imagePath} alt={`${title} icon`} style={styles.icon} />
+      </div>
+      <h3 style={styles.title}>{title}</h3>
+      <p style={styles.description}>{description}</p>
+    </div>
+  );
+};
+
+const styles = {
```
Comment (Saumya40-codes) [inline src/components/InformationBoxComponent/InformationBoxComponent.jsx]: You can move all this to `src/components/InformationBoxComponent/InformationBox.module.css` file       The style you write in this is same as what's been used in css, `.classname`

then do `import styles from ./InformationBox.module.css` in this file (`src/components/InformationBoxComponent/InformationBoxComponent.jsx`)

**Comment by vrunda2005** [comment]
Updated information box styling module.css.

File: src/components/InformationBoxComponent/InformationBoxComponent.jsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styles from "./InformationBoxComponentStyles";
```
Comment (Saumya40-codes) [inline src/components/InformationBoxComponent/InformationBoxComponent.jsx]: this gives error upon compiling
should be `import styles from "./InformationBox.module.css"`

File: src/components/InformationBoxComponent/InformationBoxComponent.jsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+const InformationBox = ({ title, imagePath, description }) => {
+  return (
+    <div style={styles.container}>
```
Comment (Saumya40-codes) [inline src/components/InformationBoxComponent/InformationBoxComponent.jsx]: here(in the whole file) it should be `className={styles.container}` only and not `style={styles.container}>` 


The way of using .module.css and .css is just same, in css its just `.classname` and when we import styles from .module.css it is `styles.classname`

**Comment by vrunda2005** [comment]
yes changes are updated now.

File: src/components/InformationBoxComponent/InformationBoxComponent.jsx (hunk 2 above)
Comment (Saumya40-codes) [reply src/components/InformationBoxComponent/InformationBoxComponent.jsx]: this left..

**Comment by vrunda2005** [comment]
solved the issue now.

**Comment by Saumya40-codes** [comment]
Perfect Now 🚀
Thanks.

**Pull request merged**